.github/workflows/test_on_push.yml

```diff
@@ -131,50 +131,50 @@ jobs:
         run: python -m nox -s integration
 
   # Skips IDAKLU module compilation for speedups, which is already tested in other jobs.
-  run_doctests:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-    name: Doctests (ubuntu-latest / Python 3.11)
-
-    steps:
-      - name: Check out PyBaMM repository
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          fetch-depth: 0
-
-      - name: Install Linux system dependencies
-        uses: awalsh128/cache-apt-pkgs-action@a6c3917cc929dd0345bfb2d3feaf9101823370ad # v1.4.2
-        with:
-          packages: graphviz pandoc
-          execute_install_scripts: true
-
-      # dot -c is for registering graphviz fonts and plugins
-      - name: Install TeXLive for Linux
-        run: |
-          sudo apt-get update
-          sudo dot -c
-          sudo apt-get install texlive-latex-extra dvipng
-
-      - name: Set up Python
-        id: setup-python
-        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
-        with:
-          python-version: 3.11
-          cache: 'pip'
-
-      - name: Set up uv
-        run: python -m pip install uv
-
-      - name: Install nox
-        run: python -m uv pip install nox[uv]
-
-      - name: Install docs dependencies and run doctests for GNU/Linux
-        run: python -m nox -s doctests
-
-      - name: Check if the documentation can be built for GNU/Linux
-        run: python -m nox -s docs
-
+  # run_doctests:
+  #   runs-on: ubuntu-latest
+  #   strategy:
+  #     fail-fast: false
+  #   name: Doctests (ubuntu-latest / Python 3.11)
+  #
+  #   steps:
+  #     - name: Check out PyBaMM repository
+  #       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+  #       with:
+  #         fetch-depth: 0
+  #
+  #     - name: Install Linux system dependencies
+  #       uses: awalsh128/cache-apt-pkgs-action@a6c3917cc929dd0345bfb2d3feaf9101823370ad # v1.4.2
+  #       with:
+  #         packages: graphviz pandoc
+  #         execute_install_scripts: true
+  #
+  #     # dot -c is for registering graphviz fonts and plugins
+  #     - name: Install TeXLive for Linux
+  #       run: |
+  #         sudo apt-get update
+  #         sudo dot -c
+  #         sudo apt-get install texlive-latex-extra dvipng
+  #
+  #     - name: Set up Python
+  #       id: setup-python
+  #       uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+  #       with:
+  #         python-version: 3.11
+  #         cache: 'pip'
+  #
+  #     - name: Set up uv
+  #       run: python -m pip install uv
+  #
+  #     - name: Install nox
+  #       run: python -m uv pip install nox[uv]
+  #
+  #     - name: Install docs dependencies and run doctests for GNU/Linux
+  #       run: python -m nox -s doctests
+  #
+  #     - name: Check if the documentation can be built for GNU/Linux
+  #       run: python -m nox -s docs
+  #
```

Comment:
This should not be commented out

Reply:
Yeah lychee can basically be moved back to the original file (just reverting the changes to that file). The `nox -s doctests` part of the commented code can be put in the `test_docs_build.yml` file. Anything redundant can be removed

Reply:
Once done, these lines can be removed entirely instead of leaving them commented out.